src/main/java/frc/robot/subsystems/Blinkin.java

```diff
@@ -0,0 +1,57 @@
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.motorcontrol.Spark;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Blinkin{
+
+
+   
+  private static Spark m_blinkin;
```

Comment:
This is assuming someone finds the Blinkin 😬

Reply:
yeah im leaving it in because its gonna be harder to re code than to remove but it is worth remembering